apps/payment/models.py

```diff
@@ -354,6 +356,17 @@ def get_from_mail(self):
     def get_to_mail(self):
         return self.user.email
 
+    def _create_fiken_sale(self, amount: float):
+        transaction_type = TransactionType.EVENT
```

Comment:
Minor nitpick: Why is this assigned here and only used one other place? Why not use it directly as a parameter as is done in the other create-function for trikom?

Reply:
Because it is not supposed to, it is supposed to use `self.transaction_type` :cold_sweat:

Reply:
Ok, in that case, it's also present in the other function for trikom.

Reply:
It is supposed to be there, since those will always be for Trikom.